1. Don't modify the following two lines!

fn main() {
    let (x, y) = (1, 2);
    let s = sum(x, y);

    assert_eq!(s, 3);

    println!("Success!");
}

fn sum(x: i32, y: i32) -> i32 { 
    x + y 
}

2. 
fn main() {
    print();
}

// Change return type to unit type ()
fn print() -> () {
    println!("Success!");
}

3.  Solve it in two way ,  DON'T let `println!` work

fn main() {
    never_return();

    println!("Failed!"); // This line will never be reached
}

fn never_return() -> ! {
    loop {} // Infinite loop
}

5.  Fill in the blank
fn main() {
    let b = true; 

    let _v = match b {
        true => 1,
        false => {
            println!("Success!");
            panic!("we have no value for `false`, but we can panic");
        }
    };

    println!("Exercise Failed if printing out this line!");
}


